app/utils/get_filtered_records.py

```diff
@@ -246,11 +246,13 @@ def grab_cols(result, columns):
 '''
 
 
-def get_paginated_records(query_dicts, sorting_key, page_index, per_page, reverse):
+def get_paginated_records(query_dicts, sorting_key, min_page_index, max_page_index, per_page, reverse):
```

Comment:
I think something we should start moving towards is parameter typing in this repo, this PR would be a nice place to start https://docs.python.org/3/library/typing.html